packages/sprite-animated/src/AnimatedSprite.ts

```diff
@@ -76,11 +77,13 @@ export class AnimatedSprite extends Sprite
 
         /**
          * `true` uses PIXI.Ticker.shared to auto update animation time.
+         *
          * @type {boolean}
          * @default true
          * @private
          */
         this._autoUpdate = autoUpdate;
+        this._isAutoUpdating = false;
```

Comment:
I was a little confused by this at first: why is another boolean needed for tracking auto-updating. But then reading the code made it more obvious. Maybe a JSDoc for this variable would be helpful:
```suggestion
        /**
         * An internal boolean to track if auto-updating is running. Differs from
         * `_playing` (which has nothing to do with auto-updates) and `_autoUpdate`
         * (which is whether it _should_ use the shared ticker). This helps with 
         * handling `autoUpdate` changes when playing or stopped.
         *
         * @type {boolean}
         * @private
         * @default false
         */
        this._isAutoUpdating = false;
```

Reply:
Updated both the variable name for clarity, and added jsdocs